homeassistant/components/philips_js/config_flow.py

```diff
@@ -53,34 +67,99 @@ async def async_step_import(self, conf: Dict[str, Any]):
             }
         )
 
+    async def _async_create_current(self):
+
+        system = self._current[CONF_SYSTEM]
+        return self.async_create_entry(
+            title=f"{system['name']} ({system['serialnumber']})",
+            data=self._current,
+        )
+
+    async def async_step_pair(self, user_input: Optional[Dict] = None):
```

Comment:
Return value typing is missing.

Reply:
If we don't want to type the items of the dict should use `dict` instead of `Dict`.